DEVELOP.rst

```diff
@@ -5,17 +5,17 @@ Developer Guide
 Setup
 =====
 
-This project uses buildout_ to set up the development environment.
+To setup the project for local development use virtualenv.
+This requires Python 3.4 or greater.
```

Comment:
@nslater python 3.4 or greater is only required for development, not for usage. should I still put it into the readme?